core/src/main/java/tech/tablesaw/filtering/deferred/QueryHelper.java

```diff
@@ -0,0 +1,155 @@
+package tech.tablesaw.filtering.deferred;
```

Comment:
It looks like there's nothing in the `filtering` package, so we could probably drop `deferred` as there's no need to differentiate

Reply:
done